### PR TITLE
Cleaning up debug messages and restored project.settings

### PR DESCRIPTION
**Pull request by MarkAYoder**
Added c-default to project.setting broke the RUN button.  The c.run script would run, but none of the variable (e.g. $file_name_base) would be defined.

### CODE REVIEW

File: .c9/project.settings.no.c-default [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (jadonk) [inline .c9/project.settings.no.c-default]: Is this file needed?

File: .c9/project.settings.c-default [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (jadonk) [inline .c9/project.settings.c-default]: Is this file needed.

File: .c9/runners/C (make).run [hunk 3]
```diff
@@ -8,5 +8,6 @@
 
     "info" : "Started $project_path: $file_name\nFile path: $file_path\nFile: $file\nArguments: $args\nFile name: $file_name\nFile extension: $file_extension\nFile base name: $file_base_name\nPackages: $packages\nProject path: $project_path\nProject name: $project_name\nHostname: $hostname\nHostname path: $hostname_path\nURL: $url\nPort: $port\nIP: $ip\nCommand: $command\nPython: $python\nPython path: $python_path",
     "env" : {},
-    "selector" : "source.c"
+    // "selector" : "source.c"
```
Comment (jadonk) [inline .c9/runners/C (make).run]: Let's just delete it. I think version control is our friend here.

File: PocketBeagle/pru/Makefile [hunk 4]
```diff
@@ -0,0 +1 @@
+include /var/lib/cloud9/common/Makefile
```
Comment (jadonk) [inline PocketBeagle/pru/Makefile]: Is this needed if we are using the runner? I'd rather the runner use -f. Needing this file is confusing.

File: common/prugpio.h [hunk 5]
```diff
@@ -54,11 +54,15 @@
...
 #define USR3 (1<<24)
-// The define a couple of GPIO pin addresses
+// The define a couple of GPIO pin addresses on Black
 // GPIO1
 #define P9_14 (1<<18)
 #define P9_16 (1<<19)
 
+// The define a couple of GPIO pin addresses on Pocket
+// GPIO1
+#define P1_32 (1<<10)
```
Comment (jadonk) [inline common/prugpio.h]: There are a number of pins that boot-up to PRU inputs. I'd guess they all should be defined at some point.

File: common/prugpio.h [hunk 6]
```diff
@@ -69,6 +73,16 @@
 #define P9_91   (1<<6)
 #define P9_25   (1<<7)
 
+// R30 output bits on pru0 on Pocket
+#define P1_36   (1<<0)
```
Comment (jadonk) [inline common/prugpio.h]: oh, wait, here are more.

File: common/prugpio.h (hunk 5 above)
Comment (jadonk) [reply common/prugpio.h]: Why is this one separate from the others?

File: PocketBeagle/pru/Makefile (hunk 4 above)
Comment (jadonk) [reply PocketBeagle/pru/Makefile]: Looking, it looks like you left my runner, but renamed it. or, maybe I forgot to push. Anyway, I guess you added this to make things run easier from the CLI and I'm cool with that. ;-)

File: .c9/project.settings.no.c-default (hunk 1 above)
Comment (MarkAYoder) [reply .c9/project.settings.no.c-default]: Nope, done

File: .c9/project.settings.c-default (hunk 2 above)
Comment (MarkAYoder) [reply .c9/project.settings.c-default]: Ditto.  Deleted

File: common/prugpio.h (hunk 5 above)
Comment (MarkAYoder) [reply common/prugpio.h]: It's for the Pocket.

Comment (MarkAYoder) [reply common/prugpio.h]: Yes, the other pins should be in there too.

File: common/prugpio.h (hunk 6 above)
Comment (MarkAYoder) [reply common/prugpio.h]: Again, for the Pocket.

File: .c9/runners/C (make).run (hunk 3 above)
Comment (MarkAYoder) [reply .c9/runners/C (make).run]: Sounds good.

File: PocketBeagle/pru/Makefile (hunk 4 above)
Comment (MarkAYoder) [reply PocketBeagle/pru/Makefile]: Yes, I renamed c.run to C (make).run and things started working.  I don't know why, but it works.

I like have the cli option too.
